templates/partial/search_result.html

```diff
@@ -0,0 +1,51 @@
+{% load markdown %}
+{% for result in results %}
+
+<div class="govuk-grid-row" id="search_result">
+    <div class="govuk-grid-column-full">
+      <h3 class="govuk-heading-m govuk-!-margin-bottom-2">
+          <a href="{% url 'home:details' %}?id={{result.id}}" class="govuk-link">{{result.name}}</a>
```

Comment:
We should refactor the url generation to pass the id into the function automatically.